.github/workflows/conda_and_docs_build.yml

```
@@ -33,92 +31,76 @@ on:
     - 'scripts/**'
     - 'NOTICE.txt'
     - 'README.md'
-
   pull_request:
-    branches: [ master ]
+    branches: [master]
     paths-ignore:
     - 'CHANGELOG.md'
     - 'CITATION.cff'
     - 'LICENSE'
     - 'scripts/**'
     - 'NOTICE.txt'
     - 'README.md'
-
 jobs:
   conda_build:
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v3.1.0
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 0
     - name: conda-build
       uses: paskino/conda-package-publish-action@v1.4.4
       with:
-        subDir: 'recipe'
-        channels: '-c conda-forge -c intel -c astra-toolbox -c ccpi'
+        subDir: recipe
+        channels: -c conda-forge -c intel -c astra-toolbox -c ccpi
         convert_win: false
         convert_osx: false
         test_pyver: 3.9
         test_npver: 1.22
     - name: Upload artifact of the conda package.
-      uses: actions/upload-artifact@v3.1.1
+      uses: actions/upload-artifact@v3
       with:
         name: cil-package
         path: recipe/linux-64/cil*
   docs_build:
     needs: conda_build
     runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v3.1.0
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - name: change directory
-      run: |
-        ls
+    - run: ls
     - name: Download artifact of the conda package.
-      uses: actions/download-artifact@v3.0.1
+      uses: actions/download-artifact@v3
       with:
-        name: 'cil-package'
+        name: cil-package
         path: 'conda_package'
     - uses: conda-incubator/setup-miniconda@v2
       with:
         python-version: 3.9
     - uses: lauramurgatroyd/build-sphinx-action@v0.1.4
       with:
-        DOCS_PATH: 'docs'
-        CONDA_BUILD_ENV_FILEPATH: 'docs/docs_environment.yml'
-        ARTIFACT_NAME: 'DocumentationHTML'
-        PACKAGE_FOLDER_PATH: 'conda_package'
-        PACKAGE_NAME: 'cil'
-        PACKAGE_CONDA_CHANNELS: 'conda-forge -c intel -c astra-toolbox -c ccpi'
-        BUILD_SUBDIR_NAME: 'nightly'
+        DOCS_PATH: docs
+        CONDA_BUILD_ENV_FILEPATH: docs/docs_environment.yml
+        ARTIFACT_NAME: DocumentationHTML
+        PACKAGE_FOLDER_PATH: conda_package
+        PACKAGE_NAME: cil
+        PACKAGE_CONDA_CHANNELS: conda-forge -c intel -c astra-toolbox -c ccpi
```

again removing astra-toolbox
```suggestion
        PACKAGE_CONDA_CHANNELS: conda-forge -c intel -c ccpi
```